server/lib/template/helpers/finance.js

```diff
@@ -30,6 +30,26 @@ function currency(value = 0, currencyId) {
   return new Handlebars.SafeString(accountingjs.formatMoney(value, fmt));
 }
 
+/**
+ * @function currencyWithPosition
+ * @desc format value with a given decimal position
+ * @todo merge this function with the currency function, since actually
+ * setting a position as third param in the currency function cause 0 decimal
+ * when nothing is defined as position we are using this currencyWithPosition
+ * @param {number} value the value to format
+ * @param {number} currencyId the id of the currency
+ * @param {number} digit the precision needed
+ * @returns {string}
+ */
+function currencyWithPosition(value = 0, currencyId, digit) {
+  const fmtNum = Number(currencyId);
+  // if currencyId is not defined, defaults to USD.
+  const fmt = fmtNum > 0 ? FORMATS[fmtNum] : USD_FMT;
+  const format = JSON.parse(JSON.stringify(fmt));
+  format.precision = digit;
+  return new Handlebars.SafeString(accountingjs.formatMoney(value, format));
+}
+
```

Comment:
I like this approach.  However, I wonder if this functionality already exists in the code somewhere -- OR whether it should be moved to some utility class so it can be used in other areas of Bhima?

Reply:
The function is like currency, i was thinking to add another parameter in `currency` but it imply to add precision anywhere we have used currency (I didn't succeed to set the default precision when we have 3 parameters)